.gitignore

```diff
@@ -35,6 +35,7 @@ build/Release
 # Dependency directories
 node_modules/
 jspm_packages/
+.idea/
```

Comment:
@petro97 very minor point but I think this would be better in your global `.gitignore` rather than in repo?

Reply:
@samblacklock Okay, I'll remove it